qemu/tests/cfg/nvdimm.cfg

```diff
@@ -92,3 +92,17 @@
         - nvdimm_hotplug:
             del mem_devs
             target_mems = mem1
+        - nvdimm_negative:
+            type = nvdimm_negative
+            start_vm = no
+            size_mem = 8G
+            pre_command = "dd if=/dev/zero of=${nv_backend} bs=1M count=8192"
+            variants:
+                - align_max:
+                    align_mem = 4G
+                    error_msg = "the alignment \(0x100000000\) is not supported|the alignment \(0x100000000\) exceeds the expected maximum alignment"
+                    ppc64le:
+                        error_msg = ""
+                - align_illegal:
+                    align_mem = 1.5G
+                    error_msg = "alignment 0x60000000 must be a power of two"
```

Comment:
on ppc, error message should be "backing store size 0x80000000 does not match 'size' option 0x200000000"

Reply:
Again, please update steps in polarion first. If the error message is different with different versions of qemu, pls make it clear too.

Reply:
size = 2G, align = 1.5G and lable_size = 128K, please see test case in the ploarion.

Reply:
The test point is align, as long as align value is as expected, other options would not affect.